### PR TITLE
Revert tradermade batch endpoint usage

### PR DESCRIPTION
**Pull request by justinkaseman**

### CODE REVIEW

**Review by austinborn — APPROVED** [review]

File: packages/sources/tradermade/src/endpoint/batch.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+export const inputParameters: InputParameters = {
+  base: {
+    aliases: ['from', 'symbol', 'market'],
+    required: true,
+    description: 'The symbol of the currency to query',
+  },
+  quote: {
+    aliases: ['to', 'market', 'convert'],
+    required: true,
```
Comment (austinborn) [inline packages/sources/tradermade/src/endpoint/batch.ts:28]: Can't have same alias "market" on both input params

**Review by austinborn — CHANGES_REQUESTED** [review]

File: packages/sources/tradermade/src/endpoint/batch.ts (hunk 1 above)
Comment (justinkaseman) [reply packages/sources/tradermade/src/endpoint/batch.ts:28]: Woah, nice catch.

Comment (austinborn) [reply packages/sources/tradermade/src/endpoint/batch.ts:28]: Must've missed it the first time though...

Comment (justinkaseman) [reply packages/sources/tradermade/src/endpoint/batch.ts]: Yeah that has been in there for ~4 months :sweat_smile:

Comment (justinkaseman) [reply packages/sources/tradermade/src/endpoint/batch.ts]: https://app.shortcut.com/chainlinklabs/story/26519/unit-test-for-duplicate-input-parameter-aliases

**Review by austinborn — APPROVED** [review]

**Review by austinborn — APPROVED** [review]

**Pull request merged**